ApiDoctor.Validation/Json/JsonSchema.cs

```diff
@@ -442,7 +442,7 @@ private PropertyValidationOutcome ValidateProperty(ParameterDefinition inputProp
                         }
                         else if (null == inputProperty.Type.CustomMembers)
                         {
-                            issues.Error(ValidationErrorCode.NoCustomMembersFound, $"Property '{inputProperty.Name}' is of type Custom but has no custom members.");
+                            //issues.Error(ValidationErrorCode.NoCustomMembersFound, $"Property '{inputProperty.Name}' is of type Custom but has no custom members.");
```

Comment:
add a comment saying this is temporarily disabled and put a link to the bug number (ideally github issue) assigned to me to investigate it